### ISSUE TITLE
set  the publish Configure through API？

### ISSUE THREAD

**Issue by yuxingwuhen** [AlexxIT/go2rtc#818]
Hi,
I don't find the API interface for setting the publish config in the API documentation. Is there  relevant interface? If not, are there any plans to add  in the future?

**Comment by AlexxIT** [comment]
```
POST /api/streams?src={stream_name}&dst=rtmp...
```

**Comment by yuxingwuhen** [comment]
yes,  i had been try this api  such as  "http://192.168.100.63:1984/api/streams?src=33&dst=rtmp%3A%2F%2F192.168.100.246%2Flive%2Fsrs63_33%2Fabc63_33".
there is no response  and the config is not changed.

**Comment by AlexxIT** [comment]
The configuration does not change when this API is called. The answer is blank if there are no errors
